### ISSUE TITLE
Renaming can delete the app or make it so you can't deploy anymore

### ISSUE THREAD

**Issue by solomon23** [caprover/caprover#490]
I ran into two issues with app renaming.

Renaming an app after enabling HTTPS
1. Create a new app
2. enable https
3. deploy app via github url /ssh key
4. rename the app

Notice an error pops up and the app is deleted

Renaming an app after deploying
1. Create a new app
2. Deploy app via github url /ssh key
3. Rename the app
4. Go back to the deployment tab and Notice the build logs is empty
5. Click `Force Build`

Note the build logs never show anything and it doesn't look like it ever builds/deploys

Here are some caprover logs

```
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:51:40.710 pm    Error: App (lithe-rename) could not be found. Make sure that you have created the app.
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | Error: App (lithe-rename) could not be found. Make sure that you have created the app.
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at Function.createError (/usr/src/app/built/api/ApiStatusCodes.js:7:16)
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at /usr/src/app/built/datastore/AppsDataStore.js:208:38
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at process._tickCallback (internal/process/next_tick.js:68:7)
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | POST /api/v2/user/apps/appDefinitions/rename 200 42.243 ms - 121
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | (node:25) UnhandledPromiseRejectionWarning: Error: Cannot force SSL without at least one SSL-enabled domain!
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at Function.createError (/usr/src/app/built/api/ApiStatusCodes.js:7:16)
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at /usr/src/app/built/datastore/AppsDataStore.js:47:42
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at process._tickCallback (internal/process/next_tick.js:68:7)
.
.
.

captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:29.428 pm    Renaming app started: From lithe-stuff To lithe-stuff-stuff 
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:29.428 pm    Renaming app: lithe-stuff
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:29.431 pm    Check if service is running: srv-captain--lithe-stuff
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:29.513 pm    Ensure service inited and Updated for: lithe-stuff-stuff
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:29.515 pm    Check if service is running: srv-captain--lithe-stuff-stuff
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:29.530 pm    Creating service srv-captain--lithe-stuff-stuff with default image, we will update image later
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:29.538 pm    Updating service srv-captain--lithe-stuff-stuff with image img-captain-lithe-stuff:1
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | GET /api/v2/user/apps/appData/lithe-web2 304 1.280 ms - -
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:31.564 pm    Updating Load Balancer
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:31.565 pm    Locking NGINX configuration reloading...
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:31.616 pm    SUCCESS: UNLocking NGINX configuration reloading...
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:31.616 pm    sendReloadSignal...
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:31.620 pm    Kill HUP Container: a5229a799870f562870f5f6835df0c4729b0a1ae2d65ef7a3342ddda84ac185b
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:31.709 pm    AppName is renamed
.
.
.

captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:47.005 pm    Error: App (lithe-stuff) could not be found. Make sure that you have created the app.
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | Error: App (lithe-stuff) could not be found. Make sure that you have created the app.
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at Function.createError (/usr/src/app/built/api/ApiStatusCodes.js:7:16)
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at /usr/src/app/built/datastore/AppsDataStore.js:208:38
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at process._tickCallback (internal/process/next_tick.js:68:7)
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | August 25th 2019, 6:55:47.007 pm    Error: Something went wrong during trigger build. Cannot extract app information from the payload.
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | Error: Something went wrong during trigger build. Cannot extract app information from the payload.
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at /usr/src/app/built/routes/user/apps/webhooks/WebhooksRouter.js:52:19
captain-captain.1.iy5i46pvkzip@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by SergioDonati** [comment]
I have same problem.
Renaming an app has completely deleted the app.
This is a big issue.

**Comment by githubsaturn** [comment]
Could you please let me know what were the exact steps that caused the issue? Are able to reproduce the problem?

**Comment by solomon23** [comment]
I can add a few more things:
1. I'm not able to repro it when trying the steps I did before.  I'll keep trying.  The time it did happen I was given a long popup error before the app was deleted.

2. If you rename an app to an app name that exists already you won't get an error but it'll delete the old app.  I can file a separate bug for that.

**Comment by solomon23** [comment]
I got this to repro just by mucking around.  The final thing that happened was I saved the websocket support and then tried to do a rename.  The rename failed saying `App (fake-name33333) could not be found. Make sure that you have created the app.` and then the app was gone.  

The Update call is successful:
`{"status":100,"description":"Updated App Definition Saved","data":{}}`

Then the`appDefinitions` call returns the app in the list with the proper name

Then the rename call fails and returns
`{"status":1000,"description":"App (fake-name33333) could not be found. Make sure that you have created the app.","data":{}}`

Then the next `appDefinitions` call doesn't include the app I tried to rename

Refreshing the site shows it's gone.

Doing a `docker container list` doesn't show the docker container running anymore

Here are the logs

```
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:50.109 pm    Renaming app started: From fake-name3 To fake-name333 
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:50.110 pm    Renaming app: fake-name3
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:50.112 pm    Check if service is running: srv-captain--fake-name3
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:50.135 pm    Ensure service inited and Updated for: fake-name333
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:50.138 pm    Check if service is running: srv-captain--fake-name333
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:50.143 pm    Creating service srv-captain--fake-name333 with default image, we will update image later
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:50.150 pm    Updating service srv-captain--fake-name333 with image img-captain-fake-name3:1
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:52.161 pm    Updating Load Balancer
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:52.162 pm    Locking NGINX configuration reloading...
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:52.226 pm    SUCCESS: UNLocking NGINX configuration reloading...
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:52.226 pm    sendReloadSignal...
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:52.231 pm    Kill HUP Container: cce382eb5c0a8ea01fef42cf41819b7a8b87fc72b74b88944879f818d41ac829
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:52.455 pm    Verifying Captain owns domain: fake-name333.dev.lithe.net
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:53.489 pm    Enabling SSL for: fake-name333
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:53.491 pm    Enabling SSL for fake-name333.dev.lithe.net
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:30:53.499 pm    executeCommand Container: captain-certbot certbot certonly --webroot -w /captain-webroot/fake-name333.dev.lithe.net -d fake-name333.dev.lithe.net --non-interactive 
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:09.006 pm    Saving debug log to /var/log/letsencrypt/letsencrypt.log
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | Plugins selected: Authenticator webroot, Installer None
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | Obtaining a new certificate
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | Performing the following challenges:
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | http-01 challenge for fake-name333.dev.lithe.net
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | Using the webroot path /captain-webroot/fake-name333.dev.lithe.net for all unmatched domains.
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | Waiting for verification...
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | Cleaning up challenges
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | 
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | IMPORTANT NOTES:
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |  - Congratulations! Your certificate and chain have been saved at:
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |    /etc/letsencrypt/live/fake-name333.dev.lithe.net/fullchain.pem
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |    Your key file has been saved at:
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |    /etc/letsencrypt/live/fake-name333.dev.lithe.net/privkey.pem
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |    Your cert will expire on 2019-11-30. To obtain a new or tweaked
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |    version of this certificate in the future, simply run certbot
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |    again. To non-interactively renew *all* of your certificates, run
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |    "certbot renew"
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |  - If you like Certbot, please consider supporting our work by:
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | 
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |    Donating to ISRG / Let's Encrypt:   https://letsencrypt.org/donate
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |    Donating to EFF:                    https://eff.org/donate-le
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | 
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | 
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:09.016 pm    Updating Load Balancer
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:09.016 pm    Locking NGINX configuration reloading...
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:09.051 pm    SUCCESS: UNLocking NGINX configuration reloading...
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:09.051 pm    sendReloadSignal...
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:09.055 pm    Kill HUP Container: cce382eb5c0a8ea01fef42cf41819b7a8b87fc72b74b88944879f818d41ac829
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:09.133 pm    AppName is renamed
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | POST /api/v2/user/apps/appDefinitions/rename 200 19024.738 ms - 59
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | GET /api/v2/user/apps/appDefinitions 200 4.709 ms - 32090
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | GET /api/v2/user/apps/appDefinitions 304 4.067 ms - -
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | GET /api/v2/user/apps/appDefinitions 304 3.472 ms - -
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:21.992 pm    Updating app started: fake-name333
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:22.003 pm    Ensure service inited and Updated for: fake-name333
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:22.004 pm    Check if service is running: srv-captain--fake-name333
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:22.008 pm    Service is already running: srv-captain--fake-name333
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:22.009 pm    Updating service srv-captain--fake-name333 with image img-captain-fake-name3:1
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:24.021 pm    Updating Load Balancer
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:24.021 pm    Locking NGINX configuration reloading...
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:24.048 pm    SUCCESS: UNLocking NGINX configuration reloading...
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:24.049 pm    sendReloadSignal...
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:24.052 pm    Kill HUP Container: cce382eb5c0a8ea01fef42cf41819b7a8b87fc72b74b88944879f818d41ac829
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:24.118 pm    Updating Load Balancer
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:24.119 pm    Locking NGINX configuration reloading...
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:24.164 pm    SUCCESS: UNLocking NGINX configuration reloading...
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:24.169 pm    sendReloadSignal...
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:24.176 pm    Kill HUP Container: cce382eb5c0a8ea01fef42cf41819b7a8b87fc72b74b88944879f818d41ac829
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:24.246 pm    AppName is updated: fake-name333
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | POST /api/v2/user/apps/appDefinitions/update 200 2254.207 ms - 69
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | GET /api/v2/user/apps/appDefinitions 200 3.271 ms - 32088
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:28.458 pm    Renaming app started: From fake-name333 To fake-name33333 
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:28.458 pm    Renaming app: fake-name333
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:28.461 pm    Check if service is running: srv-captain--fake-name333
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:28.479 pm    Ensure service inited and Updated for: fake-name33333
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | September 1st 2019, 8:31:28.480 pm    Error: App (fake-name33333) could not be found. Make sure that you have created the app.
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | Error: App (fake-name33333) could not be found. Make sure that you have created the app.
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at Function.createError (/usr/src/app/built/api/ApiStatusCodes.js:7:16)
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at /usr/src/app/built/datastore/AppsDataStore.js:208:38
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at process._tickCallback (internal/process/next_tick.js:68:7)
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | POST /api/v2/user/apps/appDefinitions/rename 200 24.513 ms - 123
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | (node:25) UnhandledPromiseRejectionWarning: Error: Cannot force SSL without at least one SSL-enabled domain!
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at Function.createError (/usr/src/app/built/api/ApiStatusCodes.js:7:16)
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at /usr/src/app/built/datastore/AppsDataStore.js:47:42
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    |     at process._tickCallback (internal/process/next_tick.js:68:7)
captain-captain.1.vbo4jqazyd6q@caprover-ubuntu-s-1vcpu-1gb-sfo1-01    | (node:25) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 100)
```

**Comment by SergioDonati** [comment]
I can confirm that is the same that happend to me.
Thanks @solomon23 for able to reproduce the issue cause i'm unable now.

**Comment by githubsaturn** [comment]
Fixed in https://github.com/caprover/caprover/pull/499 and available as edge ATM - will be released under 1.6 soon
